index.html

```diff
@@ -69,5 +69,167 @@ <h1>
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+     <div class="multi-stories">
+      <div class="grayBg"></div>
+      <div class="txt inter-font">
+         <h2>Multi-Post Stories</h2>
+         <h2>Gain+Glory</h2>
+         <ul class="projectBTNs inter-font">
+             <li> <button>Ruby on rails</button></li>
+             <li><button>css</button></li>
+             <li><button>javascript</button></li>
+             <li><button>html</button></li>
+         </ul>
+         <button class="seeProject inter-font">See Project</button>
+      </div>
+     </div>
+    </section>
+    <section class="aboutMe headline">
+      <h1 >About me</h1>
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="seeProject inter-font">Get My Resume</button>
+      <ul class="skills">
+        <li>
+          <img src="/assets/images/language.svg" alt="language Image ">
+          <a href="#" class="inter-font">Language</a>
+          <ul class="SkillsBTNs inter-font">
+            <li>
+              <button>Javascript</button>
+            </li>
+            <li>
+              <button>Ruby</button>
+            </li>
+            <li>
+              <button>Html</button>
+            </li>
+            <li>
+              <button>CSS</button>
+            </li>
+          </ul>
+        </li>
+      </ul>
```

Comment:
- [ ] I would suggest you stick as much as possible to the design. As you see the logo is bigger in size so kindly increase the size.
- [ ] The buttons have a black border please remove it and the buttons are flowing horizontally. please adjust them
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/185515333-7d7a43a8-0476-46b2-9fd0-fa45019c843a.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/185515347-f3718ad3-70d6-4458-a1fb-da1866c04719.png)

Reply:
increse the image size

Reply:
remove the border